tools/names.js

```diff
@@ -0,0 +1,435 @@
+const _ = require('underscore');
+const cheerio = require('cheerio');
+const fs = require('fs');
+const path = require('path');
+
+// Run this on the output of something like:
+// $ wget --mirror --convert-links --adjust-extension --no-parent http://www.draftexpress.com
+const folder = '/media/external/www.draftexpress.com/profile';
+
+const upperCaseFirst = str => str.charAt(0).toUpperCase() + str.slice(1);
+
+const getName = (name, file) => {
+    name = name.trim().replace(/\s\s+/g, ' '); // Condense whitespace to just single spaces
+    if (name === '') {
+        throw new Error(`No name found in ${file}, probably it is not a valid player page`);
+    }
+
+    // Handles rare first names with spaces manually, along with Nene having no last name and some typos
+    const nameFixes = {
+        'Billy Ray Bates': ['Billy Ray', 'Bates'],
+        'Hot Rod Williams': ['Hot Rod', 'Williams'],
+        'J. Robert Merritt': ['J. Robert', 'Merritt'],
+        Nene: ['Nene', ''],
+        'St. Paul Latham': ['St. Paul', 'Latham']
+    };
```

Comment:
are you sure you want to write specific exceptions for a few names that couldn't be parsed?

Also: I think Nene should be set to lastname, so that the draft scouting page would show "Nene" instead of "N."